barretenberg/cpp/src/barretenberg/dsl/acir_format/serde/witness_stack.hpp

```diff
@@ -1,49 +1,141 @@
 #pragma once
 
 #include "bincode.hpp"
+#include "msgpack.hpp"
 #include "serde.hpp"
 
-namespace WitnessStack {
+namespace Witnesses {
+struct Helpers {
+    static std::map<std::string, msgpack::object const*> make_kvmap(msgpack::object const& o, std::string name)
```

Comment:
string params should be `const std::string&` otherwise you force a copy

Reply:
this function is not templated so if you have a witness_stack.cpp, you should just declare it here, and define the body in the cpp. If you don't have one, I guess its ok...

Reply:
I thought about that, but it's just a single file at the moment.